src/test/scala/com/fulcrumgenomics/vcf/MakeTwoSampleMixtureVcfTest.scala

```diff
@@ -0,0 +1,136 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.vcf
+
+import com.fulcrumgenomics.testing.{UnitSpec, VariantContextSetBuilder}
+import htsjdk.variant.variantcontext.Allele
+import htsjdk.variant.vcf.VCFFileReader
+
+import scala.collection.JavaConversions.iterableAsScalaIterable
+
+class MakeTwoSampleMixtureVcfTest extends UnitSpec {
+  private val builder = new VariantContextSetBuilder(sampleNames = List("s1", "s2"))
+  private val (_A, _C, _G, _T, _N) = ("A", "C", "G", "T", Allele.NO_CALL_STRING)
+
+  def addVariant(pos: Int, refAllele: String, s1Allele1: String, s1Allele2: String, s2Allele1: String, s2Allele2: String) = {
+    val alleles = List(refAllele) ++ Set(s1Allele1, s1Allele2, s2Allele1, s2Allele2).filterNot(_ == refAllele).filterNot(_ == _N)
+    builder.addVariant(start=pos, variantAlleles=alleles, sampleName=Some("s1"), genotypeAlleles=List(s1Allele1, s1Allele2))
+    builder.addVariant(start=pos, variantAlleles=alleles, sampleName=Some("s2"), genotypeAlleles=List(s2Allele1, s2Allele2))
+  }
+
+  addVariant(10, _A, _A, _A, _A, _A)  // Monomorphic, should not come out
+  addVariant(20, _A, _A, _C, _A, _C)
+  addVariant(30, _A, _C, _C, _A, _A)
+  addVariant(40, _A, _A, _A, _C, _C)
+  addVariant(50, _A, _C, _C, _C, _A)
+  addVariant(60, _A, _C, _C, _C, _C)
```

Comment:
What do these do `addVariant(?, _A, _C, _C, _C, _T)`,  `addVariant(?, _A, _A, _C, _C, _C)`,  `addVariant(?, _A, _C, _C, _G, _G)`?  I think there are some more cases to test.  Basically, how do I get a mixture of two non-reference alternate alleles?